### PR TITLE
add Ticket model

### PR DESCRIPTION
**Pull request by gazayas**
### Overview:概要
I added Ticket. It's a child of Event.

### Technical changes:技術的変更点
None.

### Impact point:変更に関する影響箇所
Association with the Event model.

### Change of UserInterface:UIの変更
I changed the layers of rspec from "validation" to "association", "presence", and "numericality"

Please let me know if this is ok. Sorry there's no controller or request test yet.
Thank you!

### CODE REVIEW

File: spec/models/community_spec.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 require "rails_helper"
 
 RSpec.describe Community, type: :model do
-  describe 'validation' do
+  describe 'association' do
```
Comment (murajun1978) [inline spec/models/community_spec.rb]: 👍

File: spec/models/event_spec.rb [hunk 2]
```diff
@@ -1,13 +1,15 @@
 require "rails_helper"
 
 describe Event do
-  describe 'validation' do
+  describe 'association' do
```
Comment (murajun1978) [inline spec/models/event_spec.rb]: 👍

**Comment by gazayas** [comment]
😃

**Comment by murajun1978** [comment]
FYI: https://github.com/shinosakarb/tebukuro/issues/44#issue-191491738

Could you add `sale_starts_at` and `sale_ends_at` to ticket table ?

If the attributes name are incorrect please correct these.

**Comment by gazayas** [comment]
Ok, no problem.

Those names make sense to me. They're similar to the event model which is nice, so I'll use those.

I'll try to push changes soon

**Comment by murajun1978** [comment]
merged 🎉

**Comment by gazayas** [comment]
Thank you!

**Pull request merged**